Ray.jl/test/runtime.jl

```diff
@@ -0,0 +1,39 @@
+@testest "control logs_dir" begin
+    @testest "defaul logs_dir" begin
+        # we're running this after `init` has been called, so logs should exist already:
+        logs = readdir("/tmp/ray/session_latest/logs"; join=true)
+        @test any(contains("julia-core-driver"), logs)
+    end
+
+    @testset "custom logs_dir" begin
+        mktempdir() do logs_dir
+            cp("/tmp/ray/session_latest/logs/raylet.out", joinpath(logs_dir, "raylet.out"))
+            code = quote
+                using Ray
+                Ray.init(; logs_dir=$logs_dir)
+            end
+            cmd = `$(Base.julia_cmd()) --project=$(Ray.project_dir()) -e $code`
+            run(cmd)
```

Comment:
it might be good to test that - given a `logs_dir` - we _dont'_ get ray logs printed to stderr?

Reply:
update: _non-empty_ `logs_dir` and ENV var set to "0"